src/App.tsx

```diff
@@ -15,36 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  name = 'name',
+  length = 'length',
+}
+
+interface FilterParams {
+  field: SortField | '';
```

Comment:
The `field` property in the `FilterParams` interface is allowed to be an empty string. Ensure that this behavior is intentional and aligns with the application's requirements. If the field is empty, the goods will not be sorted, which might be the desired behavior.